DOCS/man/vo.rst

```diff
@@ -364,19 +364,58 @@ Available video output drivers are:
         Use 256 colors - for terminals which don't support true color.
 
 ``sixel``
-    Sixel graphics video output driver based on libsixel that works on a
-    console that has sixel graphics enabled such as ``xterm`` or ``mlterm``.
-    Additionally some terminals have limitation on the dimensions, so may
-    not display images bigger than 1000x1000 for example. Make sure that
-    ``img2sixel`` can display images of the corresponding resolution.
-    You may need to use ``--profile=sw-fast`` to get decent performance.
+    Graphical output for the terminal, using sixels. Tested with ``mlterm`` and
+    ``xterm``.
 
     Note: the Sixel image output is not synchronized with other terminal output
     from mpv, which can lead to broken images. The option ``--really-quiet``
     can help with that, and is recommended.
 
-    ``--vo-sixel-diffusion=<algo>``
-        Selects the diffusion algorithm for dithering used by libsixel.
+    You may need to use ``--profile=sw-fast`` to get decent performance.
+
+    Note: at the time of writing, ``xterm`` does not enable sixel by default -
+    launching it as ``xterm -ti 340`` is one way to enable it. Also, ``xterm``
+    does not display images bigger than 1000x1000 pixels by default.
+
+    To render and align sixel images correctly, mpv needs to know the terminal
+    size both in cells and in pixels. By default it tries to use values which
+    the terminal reports, however, due to differences between terminals this is
+    an error-prone process which cannot be automated with certainty - some
+    terminals report the size in pixels including the padding - e.g. ``xterm``,
+    while others report the actual usable number of pixels - like ``mlterm``.
+    Additionally, they may behave differently when maximized or in fullscreen,
+    and mpv cannot detect this state using standard methods.
+
+    Sixel size and alignment options:
+
+    ``--vo-sixel-cols=<columns>``  ``--vo-sixel-rows=<rows>`` (default: 0)
+        Specify the terminal size in character cells, otherwise (0) read it
+        from the terminal, or fall back to 80x25. Note that mpv doesn't use the
+        the last row with sixel because this seem to result in scrolling.
+
+    ``--vo-sixel-width=<width>``  ``--vo-sixel-height=<height>`` (default: 0)
+        Specify the available size in pixels, otherwise (0) read it from the
+        terminal, or fall back to 320x240. Other than excluding the last line,
+        the height is also further rounded down to a multiple of 6 (minimal
```

Comment:
"minimal" should be removed (that's my own mistake). Sixel unit height is always exactly 6.

Reply:
Have removed the "minimal" word.